code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -3501,10 +3501,10 @@
 		name = "contaminated food E.Coli"
 		id = "contaminated food E.Coli"
 		result = "e.coli"
-		required_reagents = list("sewage" = 5, "cholesterol" = 4)
+		required_reagents = list("sewage" = 5, "cholesterol" = 1)
 		result_amount = 1
 		instant = 0 // bacteria needs some time to grow in this medium
-		reaction_speed = 0.25
+		reaction_speed = 1
```

Comment:
messes with ratios, and if changing one recipe why would you not change the others

Reply:
Reverted this change.